docs/src/pages/style/color/color.md

```diff
@@ -53,16 +53,10 @@ import { createMuiTheme } from '@material-ui/core/styles';
 const theme = createMuiTheme({
   palette: {
     primary: {
-      light: '#757ce8',
       main: '#3f50b5',
-      dark: '#002884',
-      contrastText: '#fff',
```

Comment:
Do we document this feature somewhere else?

Reply:
It's covered in detail the "Themes" section, but we could add a note / cross reference here.

Reply:
Alright, then the only advantage is to illustrate how to use the output of material.io/tools/color/.